docs/plugins/repository-s3.asciidoc

```diff
@@ -388,3 +388,8 @@ To run test:
 mvn -Dtests.aws=true -Dtests.config=/path/to/config/file/elasticsearch.yml clean test
 ----
 
+= AWS VPC Bandwidth Problems
```

Comment:
Can you write something like:

```
[[repository-s3-aws-vpc]]
[float]
==== AWS VPC Bandwidth Settings
```

Because the full documentation won't render correctly otherwise